commands/test.js

```diff
@@ -49,7 +49,8 @@ function resolveJestDefaultEnvironment(name) {
 	const jestDir = path.dirname(resolve.sync('jest', {basedir: __dirname}));
 	const jestCLIDir = path.dirname(resolve.sync('jest-cli', {basedir: jestDir}));
 	const jestConfigDir = path.dirname(resolve.sync('jest-config', {basedir: jestCLIDir}));
-	return resolve.sync(name, {basedir: jestConfigDir});
+	const {TestEnvironment} = resolve.sync(name, {basedir: jestConfigDir});
+	return TestEnvironment;
```

Comment:
Why did you make this change?
I don't think it's necessary.

Reply:
Please refer to https://github.com/jestjs/jest/blob/main/packages/jest-environment-node/src/index.ts#L60.
It still exports the class itself by default.

Reply:
ok, I will revert this code.